stable/awsebscsiprovisioner/values.yaml

```
@@ -26,32 +26,31 @@ registrar:
       repository: "quay.io/k8scsi/csi-node-driver-registrar"
       tag: "v1.1.0"
 
-
 provisioner:
   # True if enable volume scheduling for dynamic volume provisioning
   enableVolumeScheduling: false
   image:
     repository: "quay.io/k8scsi/csi-provisioner"
-    tag: "v1.3.0"
+    tag: "v1.4.0"
```

```suggestion
    tag: "v1.5.0"
```
also already in here: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/aws-ebs-csi-driver/values.yaml#L15

support wise it is okay: https://kubernetes-csi.github.io/docs/external-provisioner.html

can confirm it is the latest cri-provisioner image